packages/next-swc/crates/next-dev-tests/tests/integration.rs

```diff
@@ -173,8 +173,6 @@ fn test_skipped_fails(resource: PathBuf) {
 async fn run_test(resource: PathBuf) -> JestRunResult {
     register();
 
-    let is_debug_start = *DEBUG_START;
```

Comment:
I would like to keep that command...

But I can also readd it next time I need it

Reply:
Why does the 127.0.0.1:3000 address matter for debugging?

Reply:
It's helpful to always have the same port